extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -15,43 +15,116 @@
  */
 
 import {isExperimentOn} from '../../../src/experiments';
-import {getService} from '../../../src/service';
-import {dev} from '../../../src/log';
+import {xhrFor} from '../../../src/xhr';
+import {viewerFor} from '../../../src/viewer';
+import {dev, user} from '../../../src/log';
 
 /** @private @const {string} */
 const TAG = 'amp-share-tracking';
 
-class AmpShareTracking extends AMP.BaseElement {
+export class AmpShareTracking extends AMP.BaseElement {
+  /**
+    * @return {boolean}
+    * @private
+    */
+  isExperimentOn_() {
+    return isExperimentOn(this.win, TAG);
+  }
+
+  /** @override */
+  isLayoutSupported(unusedLayout) {
+    return true;
+  }
 
   /** @override */
   buildCallback() {
-    this.isExperimentOn_ = isExperimentOn(this.win, TAG);
-    if (!this.isExperimentOn_) {
+    if (!this.isExperimentOn_()) {
```

Comment:
I still don't understand why we need `goog.asserts.assert` here. I don't see any other experimental extension using it.